app/webpacker/components/ContactEditProfilePage/index.jsx

```diff
@@ -56,3 +80,40 @@ export default function ContactEditProfilePage({ loggedInUserId, recaptchaPublic
     </Container>
   );
 }
+
+function ContactEditProfileOthers({ setContactSuccess, recaptchaPublicKey }) {
+  const { loggedInUserPermissions, loading } = useLoggedInUserPermissions();
+  const [person, setPerson] = useInputState();
+  const wcaId = person?.id;
+
+  if (loading) return <Loading />;
+
+  if (!loggedInUserPermissions.canRequestToEditOthersProfile) {
+    return (
+      <Message error>
+        <I18nHTMLTranslate i18nKey="page.contact_edit_profile.no_permission_error" />
+      </Message>
+    );
+  }
+
+  return (
+    <Container text>
+      <Header as="h2">{I18n.t('page.contact_edit_profile.title')}</Header>
+      <WcaSearch
```

Comment:
In relation to my other comment above, we also have a `IdWcaSearch` component (in the same file that defined `WcaSearch` which is purely ID-based and may help you here, because it seems that `person` is a dummy value whose only purpose is to store a WCA ID which you later pass down to the edit component.

Reply:
Okay, done.